ui/components/app/detected-token/detected-token.js

```diff
@@ -86,11 +91,11 @@ const DetectedToken = ({ setShowDetectedTokens }) => {
       ({ symbol, address }) => `${symbol} - ${address}`,
     );
     trackEvent({
-      event: EVENT_NAMES.TOKEN_HIDDEN,
-      category: EVENT.CATEGORIES.WALLET,
+      event: MetaMetricsEventName.TokenHidden,
+      category: MetaMetricsEventCategory.Wallet,
       sensitiveProperties: {
         tokens: tokensDetailsList,
-        location: EVENT.LOCATION.TOKEN_DETECTION,
+        location: MetaMetricsEventLocation,
```

Comment:
Looks like the property was forgotten here:
```suggestion
        location: MetaMetricsEventLocation.TokenDetection,
```